### DISCUSSION TITLE
unable to login since email no longer valid

### DISCUSSION THREAD

**Discussion by nigelrobbins** [community/community#70238] [New to GitHub]
Hello
I'm trying to login to an old github account of mine but the email I used (for device verification) was from an old company that I no longer work at.
Is there a way to change the email account or another way of logging in ?
Thanks

**Comment by itsdarkhost** [comment]
Yes, there is a way to change the email address associated with your GitHub account. However, you will need to contact GitHub support for assistance.

To do this, go to the GitHub support website: https://support.github.com/ and click the Contact us button. In the Category dropdown menu, select Account. In the Subject dropdown menu, select Changing my email address.

In the Description box, explain that you are trying to change the email address associated with your GitHub account, but you no longer have access to the old email address. Be sure to include your GitHub username in the description.

Once you have submitted your request, GitHub support will review it and contact you as soon as possible.
